dask_saturn/core.py

```diff
@@ -90,9 +92,15 @@ def __init__(
                 "as indicated in the Saturn Cloud UI. If those env vars are set, an external "
                 "connection will be automatically set up."
             )
+        if "autoclose" in kwargs:
+            shutdown_on_close = kwargs.pop("autoclose")
 
         self.settings = Settings()
 
+        # if dask-cluster is related to a prefect, shutdown_on_close is always true.
+        if self.settings.is_prefect:
+            shutdown_on_close = True
```

Comment:
@hhuuggoo can you comment on whether the user should be able to set this? My worry is that people might set it just by copy-pasting something they have seen around and not really intend to leave resources up after their work is done.

Reply:
setting autoclose=False only makes sense when there is a good way to re-use the cluster. For prefect cloud, since every run is now a new cluster, there is no way to re-use it, and setting autoclose=False doesn't accomplish anything (other than the proliferation of dask clusters)

Reply:
Ok cool cool. I am going to hardcode it to True then.